docs/built-in-bpa-rules.md

```diff
@@ -1,196 +1,3 @@
-#### Severity: 3
-
-### TA-000028: SQL servers with auditing to storage account destination should be configured with 90 days retention or higher
-Set the data retention for your SQL Server's auditing to storage account destination to at least 90 days.
-
-**Recommendation**: For incident investigation purposes, we recommend setting the data retention for your SQL Server's auditing to storage account destination to at least 90 days, in the [Microsoft.Sql/servers/auditingSettings resource properties](https://docs.microsoft.com/azure/templates/microsoft.sql/2020-11-01-preview/servers/auditingsettings?tabs=json#serverblobauditingpolicyproperties-object), using the *retentionDays* property. Confirm that you are meeting the necessary retention rules for the regions in which you are operating. This is sometimes required for compliance with regulatory standards.
-#### Severity: 3
-
-### TA-000029: Azure API Management APIs should use HTTPS only
-Set the protocols property of your Azure API Management APIs to only include HTTPS.
-
-**Recommendation**: To use encrypted protocols only, add (or update) the *protocols* property in the [Microsoft.ApiManagement/service/apis resource properties](https://docs.microsoft.com/azure/templates/microsoft.apimanagement/service/apis?tabs=json), to only include HTTPS. Allowing any additional protocols (e.g. HTTP, WS) is insecure.
-#### Severity: 1
-
-### TA-000030: Migrate your Classic Compute VM to ARM
-Azure supports two deployment models: Classic and Azure Resource Manager (ARM). ARM provides several security enhancements to the Classic model. You can learn more about the differences [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/deployment-models).
-
-**Recommendation**: Use ARM for your virtual machines to provide security enhancements such as: stronger access control (RBAC), better auditing, ARM-based deployment and governance, access to managed identities, access to key vault for secrets, Azure AD-based authentication and support for tags and resource groups for easier security management.
-#### Severity: 1
-
-### TA-000031: Migrate your Classic Storage Account to ARM
-Azure supports two deployment models: Classic and Azure Resource Manager (ARM). ARM provides several security enhancements to the Classic model. You can learn more about the differences [here](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/deployment-models).
-
-**Recommendation**: Use ARM for your storage accounts to provide security enhancements such as: stronger access control (RBAC), better auditing, ARM-based deployment and governance, access to managed identities, access to key vault for secrets, Azure AD-based authentication and support for tags and resource groups for easier security management.
-#### Severity: 1
\ No newline at end of file
+The rule documentation has moved [here](built-in-rules.md).
```

Comment:
```suggestion
The rule documentation has moved to [Built-In Rules](built-in-rules.md).
```

Lower case seems like a dramatically incorrect sentence, like this is obvious we mean the title of the page